### DISCUSSION TITLE
Is there a way to add a module to the end of the prompt?

### DISCUSSION THREAD

**Discussion by KailashKS** [starship/starship#2966] [Q&A]
powerlink10k zsh allows you to add less important stuff like time, ram usage and stuff to the end of the prompt. Is that possible with starship? Sorry if this seems dumb but I am just starting out with terminal customisation in general.

**Comment by chipbuster** [comment]
Yes, you can do something like that in starship (though the config can be a little irritating to deal with).

To do this, modify the [prompt order](https://starship.rs/config/#default-prompt-format) so that the appropriate modules appear at the end. Note that both memory usage and time are disabled-by-default. To enable them, add the appropriate settings to `starship.toml`, e.g. for memory usage:

```
[memory_usage]
disabled = false
```

(and similarly for time)

**Reply by KailashKS** [reply]
But how do I add it at the end though? 
I have added my starship.toml file below. I have also added an image showing my prompt. I even tried to add \t multiple times to push the time module but that just breaks on terminal resizing.

`command_timeout	= 3000
format = """[username $directory$cmd_duration$git_branch$time
$memory_usage $character [❯](bold yellow)[❯](bold red)[❯](bold green)](bold) """

add_newline = true

[line_break]
disabled = false

[character]
format = "$symbol"
vicmd_symbol = "[V](bold green)"
error_symbol = "[✗](bold red)"
success_symbol = "[✓](bold green)"

Disable the package module, hiding it from the prompt completely
[package]
disabled = true

[python]
disabled = true

[username]
format = "[$user](bold white) in"
disabled = false
show_always = true

[cmd_duration]
min_time = 500
format = "took [$duration](bold yellow)"

[memory_usage]
format = "$ram_pct"
threshold = -1
disabled = false
style = "bold dimmed green"

[directory]
format = "[$path]($style)[$read_only]($read_only_style) "
read_only = "🔒"
read_only_style = "red"
style = "bold cyan"

[git_branch]
disabled = false
format = "on [$symbol$branch]($style) "
symbol = " "
style = "bold purple"

[time]
disabled = false
format = '[$time](bold yellow) '
time_format = "%T"`

![image](https://user-images.githubusercontent.com/34036909/128567715-657ed897-d592-43f6-b3d5-7bb5a5e2d005.png)

**Reply by chipbuster** [reply]
Are you trying to add the time at the end of the prompt or aligned with the right hand side of the terminal window?

If the latter, unfortunately we don't support that at the moment (see #498 for progress on this feature).

**Reply by KailashKS** [reply]
My bad. My phrasing was wrong. I actually do want it aligned to the right hand side of the terminal window. Nice to see that there is progress on the feature and I will be looking forward to when it is complete. Thank you for your help.
